deploy.sh

```diff
@@ -342,6 +342,37 @@ function precheck {
   init_terraform
 }
 
+# -------------------------------------------------------------------------
+# Function to read sensitve data by masking with asterisk
+# -------------------------------------------------------------------------
+function read_sensitive_data {
+  stty -echo
+  #read password
+  charcount=0
+  while IFS= read -p "$prompt" -r -s -n 1 ch
+  do
+      # Enter - accept password
+      if [[ $ch == $'\0' ]] ; then
+          break
+      fi
+      # Backspace
+      if [[ $ch == $'\177' ]] ; then
+          if [ $charcount -gt 0 ] ; then
+              charcount=$((charcount-1))
+              prompt=$'\b \b'
+              value="${value%?}"
+          else
+              prompt=''
+          fi
+      else
+          charcount=$((charcount+1))
+          prompt='*'
+          value+="$ch"
+      fi
+  done
+  echo
```

Comment:
change this to `stty echo`

Reply:
After doing this change please fix the rest of the appearing issues.

1. Extra asterisk char appearing for pull-secret before entering anything (when followed by RHEL sub inputs)
2. Prompt for pull-secret not coming on newline

Reply:
@Prajyot-Parab  @yussufsh  Signed and fixed with respect to above comments. Please validate. Thanks.